polaris-react/src/components/Pagination/Pagination.scss

```diff
@@ -24,4 +24,41 @@
       }
     }
   }
+
+  &.table {
+    button {
+      #{$se23} & {
+        --pc-button-min-height: var(--p-space-6);
+
+        background-color: var(--p-color-bg-subdued);
+        min-height: var(--pc-button-min-height);
+        min-width: var(--pc-button-min-height);
+        height: var(--pc-button-min-height);
+        width: var(--pc-button-min-height);
+        padding: unset;
+
+        &:hover {
```

Comment:
Can we add a `:focus-visible` style here too?

Reply:
Nvm! Just played around with the story and the focus ring is already handled.